point_pattern.py

```diff
@@ -124,7 +125,7 @@ def check_in(point, point_list):
     point_list : list
                  in the form [point, point_1, point_2, ..., point_n]
     """
-    return None # Add the logic to check if a point is in the point list here
+    return (point in point_list) # Add the logic to check if a point is in the point list here
```

Comment:
Here you do not need to put `point in point_list` inside of the quotes (this casts the results to a tuple).  This is a style thing primarily - a code checker (linter) would throw a warning.  Obviously this still works, just a FYI.